FHIR-Converter/README.md

```diff
@@ -86,9 +115,17 @@ Currently, the following features for snippet templates editing are supported:
 
 ![snippet-templates-editing](assets/snippet-templates-editing.gif)
 
-### 5. Update the template folder
+### 7. Push templates
+
+After modifying the templates, you can save and push the templates to the private ACR. You also need to confirm whether you have logged in to ACR.
```

Comment:
..your private ACR. You must be logged in to your ACR in order to push the templates.